report_thesis/src/sections/future_work.tex

```diff
@@ -1 +1,17 @@
-\section{Future Work}\label{sec:future_work}
\ No newline at end of file
+\section{Future Work}\label{sec:future_work}
+The findings of this study open several avenues for future research.
+Firstly, for our data partitioning algorithm, described in Section~\ref{subsubsec:dataset_partitioning}, we noted that finding the optimal percentile value $p$ that minimizes extreme values in the test set, while maintaining its general representativeness, is important.
+Future work should consider methods of quantitatively assesing and finding this value.
+Such methods could include supplementary extreme value testing to the data partitioning algorithm, where after the primary evaluation, additional testing is conducted using a small, separate subset of extreme values to assess the model's performance on these critical cases.
+For example, this could involve slightly reducing the percentile value $p$ and using the extreme values that fall within this reduced range to evaluate the model.
```

Comment:
```suggestion
For example, this might involve slightly reducing the percentile value $p$ and using the extreme values that fall within this reduced range to evaluate the model's effectiveness.
```